### PR TITLE
Whitespace cleanup in bindings

### PR DESCRIPTION
**Pull request by MichalPetryka**
Same as #454, this one has a lot of changes mostly due to tabs being converted to spaces.

### CODE REVIEW

File: bindings/cpp/include/portaudiocpp/SampleDataFormat.hxx [hunk 1]
```diff
@@ -11,21 +11,21 @@ namespace portaudio
...
-	};
+    //////
+    /// @brief PortAudio sample data formats.
+    ///
+    /// Small helper enum to wrap the PortAudio defines.
+    //////
+    enum SampleDataFormat
+    {
+        INVALID_FORMAT  = 0,
+        FLOAT32         = paFloat32,
```
Comment (philburk) [inline bindings/cpp/include/portaudiocpp/SampleDataFormat.hxx:22]: The expand tool seems to handle tabs in the middle of lines very well.

**Review by philburk — APPROVED** [review]

File: bindings/cpp/include/portaudiocpp/SampleDataFormat.hxx (hunk 1 above)
Comment (MichalPetryka) [reply bindings/cpp/include/portaudiocpp/SampleDataFormat.hxx:22]: Yeah, it has an option to only convert tabs at the start of lines, but after seeing things like this nicely converted I used it without that.

**Comment by philburk** [comment]
Note that we are planning to remove the Java and C++ bindings from PortAudio.

**Comment by MichalPetryka** [comment]
If the plan is to move them to a separate repository, cleaning them up before doing it shouldn't be an issue.

**Review by RossBencina — APPROVED** [review]

**Pull request merged**